### PR TITLE
Use platformio as build system

### PR DESCRIPTION
**Pull request by dl1com**

### CODE REVIEW

File: platformio.ini [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[platformio]
+default_envs = uno
+
+[env]
+framework = arduino
+build_flags =
+    -DAYAB_QUIET
```
Comment (dl1com) [inline platformio.ini:17]: I don't think this flag is even used in the code?

**Comment by t0mpr1c3** [comment]
Might be good to get @X-sam to have a look as she is almost certainly more familiar with PlatformIO.

**Comment by t0mpr1c3** [comment]
Is `build.sh` still used somewhere?

**Comment by dl1com** [comment]
> Is `build.sh` still used somewhere?

No, it can be removed as well

**Comment by zelatyna** [comment]
I tested this today and works like a charm
Just one thing - once I switched to branch `v1.0-platformio` I needed to update submodules

**Comment by dl1com** [comment]
> I tested this today and works like a charm Just one thing - once I switched to branch `v1.0-platformio` I needed to update submodules

Right, that is because the submodules have been moved from the "libraries" to the "lib" folder.
So please make sure to run `git submodule update --init --recursive` once this is merged.
Freshly cloned repos shouldn't be affected.

**Pull request merged**